spec/features/ineligible_levelling_up_premium_payments_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+
+RSpec.feature "Ineligible Levelling up premium payments claims" do
+  let(:eligibility) { LevellingUpPremiumPayments::Eligibility.order(:created_at).last }
+
+  scenario "When the school selected is LUP ineligible" do
+    start_levelling_up_premium_payments_claim
```

Comment:
Again maybe another PR, given it's a combined journey, should we make different helper that returns both the claims, because visiting the url should create 2 claims.